### ISSUE TITLE
How can I get Codewhisperer response list or data from the output panel

### ISSUE THREAD

**Issue by chenxuanzzy** [aws/aws-toolkit-vscode#4126]
## System details (run the `AWS: About Toolkit` command)

OS: Darwin arm64 22.6.0
Visual Studio Code extension host:  1.84.2
AWS Toolkit:  2.1.0
node: 18.15.0
electron: 25.9.2

## Question
I have been using codewhisperer while, but every time I get the response, I need to copy it on the chat, I also use AWS Toolkit Logs but can't find the response data/list, just got the question data. How can I get the response data on the output panel or something like that?

**Comment by justinmk3** [comment]
Are you trying to copy the response in Amazon Q chat? Can you clarify the steps you are trying?

**Comment by chenxuanzzy** [comment]
> Are you trying to copy the response in Amazon Q chat? Can you clarify the steps you are trying?

I'm sorry for my explanation.
I was trying to ask Amazon Q on the chat, and I also can see the AWS Toolkit Logs on the panel. 
In the panel, I just can only find the asking object
```
2023-12-06 14:39:39 [INFO]: request from tab: tab-1 coversationID: undefined request: {"conversationState":{"currentMessage":{"userInputMessage":{"content":"give me some eample code","userInputMessageContext":{"editorState":{"document":{"relativeFilePath":"/Users/cindy_chao/Develop/test/test2.ts","text":"import * as http from \"http\";\n","programmingLanguage":{"languageName":"typescript"},"documentSymbols":[]},"cursorState":{"range":{"start":{"line":0,"character":0},"end":{"line":1,"character":0}}}}}}},"chatTriggerType":"MANUAL"}}
```
can't find the response object or other data.
how can get the response log data? or where can I get the response logfile? I can't find it out.

![image](https://github.com/aws/aws-toolkit-vscode/assets/59633053/e92d4bd8-b2d2-4fee-904d-f2cf808fa9ec)

**Comment by chenxuanzzy** [comment]
> Are you trying to copy the response in Amazon Q chat? Can you clarify the steps you are trying?

**Comment by chenxuanzzy** [comment]
@justinmk3 I'm sorry. Can anyone respond to me :( ?

**Comment by justinmk3** [comment]
@chenxuanzzy In the screenshot, there is a "Copy" button which allows you to copy the answer. But I guess that isn't what you're asking for. Are you trying to inspect the raw service response? Do you want AWS Toolkit to print more data in its logs? 

I'm having trouble understanding your request.

**Comment by chenxuanzzy** [comment]
> * Are you trying to inspect the raw service response? Do you want AWS Toolkit to print more data in its logs?

Yes, This is what I want. I want AWS Toolkit logs have answers other json I can watch.

**Comment by justinmk3** [comment]
Did you try setting the AWS Toolkit log level to "debug"? In VSCode settings look for "aws log". 
![image](https://github.com/aws/aws-toolkit-vscode/assets/55561878/d104a4cd-72c7-4d21-aaaf-37289641ac7c)
